commit e3130695cec1e0fb3144e2f44040a254bebc08e4
Author: John E. Davis <jed@jedsoft.org>
Date:   Mon Feb 2 10:22:43 2015 -0500

    pre2.3.1-17: remove duplicate prototype for SLrline_get_update_client_data from slang.h

diff --git a/src/slang.h b/src/slang.h
index 7df1552..0356370 100644
--- a/src/slang.h
+++ b/src/slang.h
@@ -1624,7 +1624,6 @@ SL_EXTERN int SLrline_set_update_hook (SLrline_Type *,
 				    VOID_STAR client_data);
 /* free update_hook client_data */
 SL_EXTERN void SLrline_set_free_update_cb (SLrline_Type *, void (*)(SLrline_Type *, VOID_STAR));
-SL_EXTERN int SLrline_get_update_client_data (SLrline_Type *, VOID_STAR *);
 
 /* These functions are passed a pointer to the update_hook client_data */
 SL_EXTERN void SLrline_set_update_clear_cb(SLrline_Type *, void (*)(SLrline_Type *, VOID_STAR));
